### PR TITLE
Add limited ability to register build tasks to a site

### LINKED ISSUE #4403: create non-manual (DB login) way of enabling tasks for a site

### PR DESCRIPTION
**Pull request by drewbo**
## Changes proposed in this pull request:
- add limited ability to register and/or remove build tasks associated with a site
- needed for improved testing in dev/staging
- close #4403 

## security considerations
Adds admin ability to register build tasks for a site and also remove them

### CODE REVIEW

File: api/controllers/build-task.js [hunk 1]
```diff
@@ -69,7 +69,9 @@ module.exports = wrapHandlers({
       return res.forbidden();
     }
 
-    await task.update(body);
+    const updatedTask = { ...body, updatedAt: Date.now() };
```
Comment (apburnes) [inline api/controllers/build-task.js:72]: `updatedAt` doesn't automatically get a new timestamp on update?

File: api/admin/routers/api.js [hunk 2]
```diff
@@ -52,7 +52,10 @@ apiRouter.put('/sites/:id', AdminControllers.Site.update);
 apiRouter.get('/sites/:id/webhooks', AdminControllers.Site.listWebhooks);
 apiRouter.post('/sites/:id/webhooks', AdminControllers.Site.createWebhook);
 apiRouter.delete('/sites/:id', authorize(['pages.admin']), AdminControllers.Site.destroy);
+apiRouter.post('/sites/:id/tasks', AdminControllers.Task.addSiteBuildTask);
 apiRouter.get('/tasks', AdminControllers.Task.list);
+apiRouter.get('/tasks/types', AdminControllers.Task.listTypes);
+apiRouter.delete('/sbt/:id', authorize(['pages.admin']), AdminControllers.Task.removeSiteBuiltTask);
```
Comment (apburnes) [inline api/admin/routers/api.js:58]: Should this route be more semantic with `/site-build-tasks/:id`

File: api/controllers/build-task.js (hunk 1 above)
Comment (drewbo) [reply api/controllers/build-task.js:72]: good call, it does, I added this when I didn't see updates but it was a front-end spelling error

**Review by apburnes — APPROVED** [review]
💯

**Pull request merged**